pkg/build/gobuild_test.go

```diff
@@ -719,6 +719,11 @@ func TestGoBuild(t *testing.T) {
 		withSBOMber(fauxSBOM),
 		WithLabel("foo", "bar"),
 		WithLabel("hello", "world"),
+		WithDefaultImageConfig(&ImageConfig{Labels: map[string]string{
```

Comment:
Could you also test the per-image config values here too?

Reply:
OFC, done